README.md

```diff
@@ -284,7 +284,7 @@ In order to submit In-Person requests with [Terminal API over Cloud](https://doc
 import com.adyen.Client;
 import com.adyen.enums.Environment;
 import com.adyen.service.TerminalCloudAPI;
-import com.adyen.model.nexo.*;
+import com.adyen.terminal.*;
```

Comment:
Should we make it more clear what we should import from where? I think the `com.adyen.terminal.*` import is pretty fuzzy compared to how other services work

Reply:
Updated the imports in the readme, should we proceed with this?

Reply:
Added implicit imports